_careers/data_scientist.md

```diff
@@ -0,0 +1,44 @@
+---
+layout: default
+title: Junior Data Scientist
+location: "Prague, CZ"
+excerpt: "As a part of our Data Science team, you will get the opportunity to design, build and test models and applications that deliver analytical and predictive content directly to users, combining commercial, analytical and technological expertise. Additionally, we contribute to open-source solutions and to present at conferences and summits. Our mission is to play a key role in helping ABSA, a huge and multinational financial corporation, to become data-driven. "
+---
+Location: {{ page.location }}
+
+### About the role
+{{ page.excerpt }}
+
+### Responsibilities
+- Work closely with the business stakeholders and technology leaders
+- Design and implementation of Data Science products in various business area
+- Research of new technologies and investigation of their usability in our systems
+
+### Must Have
+- Strong problem solving skills
+- Fluent written and spoken English
+- Some experience with coding, ideally python
+- Background in computer science, engineering, physics, mathematics or equivalent
+- Be eager to learn, improve and grow together with the team
+
+
+### Desirable Skills
+- Be able to creatively solve challenges, by out of the box thinking, while having an understanding of the required business needs
```

Comment:
I think all the commas in this sentence are unnecessary but I would not stake on it.

Reply:
Ok updated + added one more requirement based on the feedback from the team

Reply:
out of the [Jankins-Box](https://en.wikipedia.org/wiki/Box%E2%80%93Jenkins_method) thinking... 😆 
sorry, just joking